### ISSUE TITLE
Can’t use git

### ISSUE THREAD

**Issue by taynpg** [ish-app/ish#1802]
I Can’t use git on iPhone8 Plus(iOS version 15.4.1)，ish version is 1.2.3 (build 298)，it will interrupt when it runs and can’t to the next step.

**Comment by amoxuk** [comment]
@taynpg 
use the aliyun mirror.
```shell
apk add git
git clone xxxx
```
it will be success.

the version of me:
>    # git --version
>    git version 2.32.1
>    # uname -a
>    Linux amoxuk 4.20.69-ish iSH 1.2.3 (298) Dec 17 2021 06:08:24 i686 Linux

**Comment by taynpg** [comment]
> 

I Don’t know why it doesn’t work. It interrupted.
![Uploading D36BC596-54B0-4CE8-9A4B-788DD0D91A58.png…]()

**Comment by taynpg** [comment]
I Don’t know why it doesn’t work. It interrupted.
![Uploading 051A38A5-571E-4B3B-886E-0666EA01A8C2.png…]()

**Comment by taynpg** [comment]
I tried to use the aliyun mirror , but it also same, It will stop at the xx% and then stop(git can been installed succeed.

**Comment by taynpg** [comment]
nokia:~# rm -rf cqm
nokia:~# git clone https://gitee.com/taynpg/cqm
Cloning into 'cqm'...
remote: Enumerating objects: 342, done.
remote: Counting objects:  46% (158/342)

Something like this.

**Comment by amoxuk** [comment]
![image](https://user-images.githubusercontent.com/26641021/169530134-8c063807-2f75-4c59-8dfc-4bb6358499f7.png)
it's ok in my phone.

**Comment by taynpg** [comment]
Thinks

**Comment by owner888** [comment]
> nokia:~# rm -rf cqm nokia:~# git clone https://gitee.com/taynpg/cqm Cloning into 'cqm'... remote: Enumerating objects: 342, done. remote: Counting objects: 46% (158/342)
> 
> Something like this.

me too, I use Ipad11 pro

**Comment by ketudb** [comment]
> me too, I use Ipad11 pro

likewise on the same ipad! using `alpine-minirootfs-3.14.1-x86`

**Comment by lingjie00** [comment]
The Git command seems to work sometimes, but sometimes the progression will get stuck as described. Restarting the App sometimes solve the problem.

**Comment by lamurian** [comment]
Have a similar problem as well, I'm currently using `4.20.69-ish`, the latest iSH shell from the App store. It's using alpine repository `v3.14` from  `https://dl-cdn.alpinelinux.org/alpine/v3.14/{main,community}`. I've tried configuring git to use a single thread (`git config --global pack.threads "1"`) and setting the maximum depth when cloning the repo but those suggestions don't seem to work. Weirdly enough, yesterday I tried using git and it worked fine.
